### ISSUE TITLE
Name collision with Cloud Foundry “Garden”

### ISSUE THREAD

**Issue by ericis** [garden-io/garden#922]
The name of this project was confusing to me at first as I thought it referred to container management from Cloud Foundry called “Garden”.
https://docs.cloudfoundry.org/concepts/architecture/garden.html

Consider renaming for this reason. Or, don’t 🙂

**Comment by baspetersdotjpeg** [comment]
Hey Eric, Thanks for pointing this out! We named the project 'Garden' because it felt like a nice and welcoming name. With a generic term such as 'Garden', it's impossible not to collide with other product names out there, and it's unfortunate that Cloudfoundry's project is in a similar space (containers, broadly speaking). We were unaware of the name collision when we first released Garden, but since what we do is fundamentally different, and we're not infringing on any trademarks (or otherwise negatively affect anyone's efforts) we'll just have to live with sometimes being confused for other projects. It's worth it for such a cool name ;)
